package.json

```diff
@@ -47,6 +47,7 @@
     "site": "bundle exec jekyll serve",
 
     "lint": "tslint -c tslint.json 'src/**/*.ts' 'test/**/*.ts'",
+    "lint:fix": "tslint -c tslint.json --fix 'src/**/*.ts' 'test/**/*.ts'",
```

Comment:
No because Travis just executes something like `npm run lint` and @domoritz suggestion was to pass the flag _locally_ into the CLI to fix the raised issues when the "ordered-imports" was turned on.

This flag enables tslint to fix (and change) the files and those changes were what I commit (cc0f4cc) and pushed. So as this was manually (and locally) executed, Travis will continue working as expected.

This makes sense to you?